mappings/net/minecraft/world/biome/Biome.mapping

```diff
@@ -154,3 +166,26 @@ CLASS net/minecraft/class_1959 net/minecraft/world/biome/Biome
 			ARG 2 weight
 			ARG 3 minGroupSize
 			ARG 4 maxGroupSize
+	CLASS class_4762 MixedNoisePoint
+		COMMENT Represents a point in a multi-dimensional cartesian plane. Mixed-noise
+		COMMENT biome generator picks the closest noise point from its selected point
+		COMMENT and choose the biome associated to that closest point. Another factor,
+		COMMENT influence, allows a farther point
+		FIELD field_22043 heat F
+		FIELD field_22044 humidity F
+		FIELD field_22045 hilliness F
```

Comment:
if we're basing names off of the biome settings then this should be `scale`

Reply:
scale? or depth?

Reply:
I see no need to be consistent with biome setting names here. These names represent our best guess on what these values are supposed to mean based on the values that different biomes have them set for. Ultimately they aren't really related to the values in biome settings, and can hold different, even contradictory values (see the badlands biome). These names are fine as they are.